### PR TITLE
fix: loosen base64 validation

### LINKED ISSUE #1308: Unpadded base64 data is not accepted in an attachment decorator
Notably, this can happen with the attachment on a did-exchange response message. The message is rejected due to the IsBase64 check failing.

### PR DESCRIPTION
**Pull request by TimoGlastra**
Fixes #1308

### CODE REVIEW

**Comment by andrewwhitehead** [comment]
Thanks!

File: packages/core/src/decorators/attachment/Attachment.ts [hunk 1]
```diff
@@ -44,7 +34,7 @@ export class AttachmentData {
    * Base64-encoded data, when representing arbitrary content inline instead of via links. Optional.
    */
   @IsOptional()
-  @IsBase64()
+  @IsString()
```
Comment (genaris) [inline packages/core/src/decorators/attachment/Attachment.ts:37]: I'm OK with this change if it fixes the problem but I'm curious about the reason it does not accept unpadded base64. Is an issue with class-validator?

**Review by genaris — APPROVED** [review]
I'm OK with this change but curious about why is it a problem for

File: packages/core/src/decorators/attachment/Attachment.ts (hunk 1 above)
Comment (TimoGlastra) [reply packages/core/src/decorators/attachment/Attachment.ts:37]: I think it only accept plain base64, not any of the variant in between base64 and base64url?

**Pull request merged**